src/gluonts/nursery/spliced_binned_pareto/run_model_example.ipynb

```diff
@@ -45,7 +45,7 @@
     "import os\n",
     "import pprint\n",
     "import random\n",
-    "from scipy import stats \n",
```

Comment:
@leica2023 could you revert the changes to notebooks? No need to run `black` on them